SimplifiedTests/NYPLBookmarkSpec+AudiobookTests.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  NYPLBookmarkSpec+AudiobookTests.swift
+//  SimplyETests
+//
+//  Created by Ernest Fan on 2022-08-24.
+//  Copyright © 2022 NYPL. All rights reserved.
+//
+
+import XCTest
+import NYPLUtilities
+import NYPLAudiobookToolkit
+@testable import SimplyE
+
+class NYPLBookmarkSpec_AudiobookTests: XCTestCase {
+  var bundle: Bundle!
+
+  override func setUpWithError() throws {
+    bundle = Bundle(for: NYPLBookmarkSpec_AudiobookTests.self)
+  }
+
+  override func tearDownWithError() throws {
+    bundle = nil
+  }
+
+  func testMakeAudiobookLocatorFromJSON() throws {
+    // preconditions: get expected values from manually reading locator on disk
+    let locatorURL = bundle.url(forResource: "valid-locator-3", withExtension: "json")!
+    let locatorData = try Data(contentsOf: locatorURL)
+    let json = try JSONSerialization.jsonObject(with: locatorData) as! [String: Any]
+    let typeValue = json[NYPLBookmarkSpec.Target.Selector.Value.locatorTypeKey] as! String
+    let title = json[NYPLBookmarkSpec.Target.Selector.Value.audiobookLocatorTitleKey] as! String
+    let audiobookId = json[NYPLBookmarkSpec.Target.Selector.Value.audiobookLocatorBookIDKey] as! String
+    let chapter = json[NYPLBookmarkSpec.Target.Selector.Value.audiobookLocatorChapterKey] as! Int
+    let part = json[NYPLBookmarkSpec.Target.Selector.Value.audiobookLocatorPartKey] as! Int
+    let duration = json[NYPLBookmarkSpec.Target.Selector.Value.audiobookLocatorDurationKey] as! Double
+    let offset = json[NYPLBookmarkSpec.Target.Selector.Value.audiobookLocatorOffsetKey] as! Double
+
+    // test: make a locator, encode it to binary, parse binary back to JSON
+    let madeLocatorString = NYPLAudiobookBookmarkFactory.makeLocatorString(title: title,
```

Comment:
since here we're testing `NYPLAudiobookBookmarkFactory.makeLocatorString`, I'm wondering if this unit test should be moved to NYPLAudiobookToolkit?

Reply:
The reason we want to implement the test here is because we need access to the sample locator/bookmark data in the `Simplified-Bookmark-Spec` repo. 
Moving these tests to `NYPLAudiobookToolkit` means we either 
- add the  `Simplified-Bookmark-Spec` repo as a dependency of `NYPLAudiobookToolkit`
- or we copy those locator/bookmark sample files to `NYPLAudiobookToolkit`
What do you think?

Reply:
I'd be fine with either option unless it really complicates the build. If it does, don't worry about it. If it's just a matter of adding gitsubmodule and/or moving the 3 new locators/bookmarks samples, I think it's worth it, because it keeps source code + related tests closer together, which is expected.